web/annotation_editor_layer_builder.css

```diff
@@ -285,6 +281,84 @@
           outline: 2px solid var(--editor-toolbar-focus-outline-color);
         }
       }
+
+      .altText {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: max-content;
+        padding-inline: 16px;
```

Comment:
Do we still want this all that "much" padding here, since it feels unnecessarily large now that it's a toolbar button?

Reply:
I don't know, it's what I read in the specs.
I raised the question to UX.